features/harmful-apis.json

```diff
@@ -0,0 +1,106 @@
+{
+    "_meta": {
+        "description": "Protection from APIs harmful for user privacy.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "site breakage"
+        }
+    },
+    "settings": {
+        "deviceOrientation": {
+            "protected": true,
+            "filterEvents": [
+                "deviceorientation",
+                "devicemotion"
+            ]
+        },
+        "GenericSensor": {
+            "protected": true,
+            "filterPermissions": [
+                "accelerometer",
+                "ambient-light-sensor",
+                "gyroscope",
+                "magnetometer"
+            ],
+            "blockSensorStart": true
+        },
+        "UaClientHints": {
+            "protected": true,
+            "highEntropyValues": {
+                "trimBrands": true,
+                "model": "",
+                "trimPlatformVersion": 2,
+                "trimUaFullVersion": 1,
+                "trimFullVersionList": 1
+            }
+        },
+        "NetworkInformation": {
+            "protected": true
+        },
+        "getInstalledRelatedApps": {
+            "protected": true,
+            "returnValue": []
+        },
+        "FileSystemAccess": {
+            "protected": true,
+            "disableOpenFilePicker": true,
+            "disableSaveFilePicker": true,
+            "disableDirectoryPicker": true,
+            "disableGetAsFileSystemHandle": true
+        },
+        "WindowPlacement": {
+            "protected": true,
```

Comment:
Yeah it was kinda on purpose, to avoid confusion between this and the upper-level `"state": "enabled"`, that we normally set per platform in `*-override.json`. I don't have a strong opinion on this though, so if you do, I'm ok with changing this

Reply:
makes sense yeah :) - I know in some recent features like Runtime Checks and Duck Player for example, we opted to represent what we considered 'sub features' in the same way (with `"enabled" | "disabled"`), eg: https://duckduckgo.github.io/content-scope-scripts/modules/Duck_Player_Overlays.html#md:remote-config

^ in that example, state: "enabled" | "disabled" is used in both the top-level `duckPlayer` feature + the individual sub-features - works the same way in Runtime Checks (the 2 most recent features).

I wouldn't personally block your PR on this (as it's easy to revert/change) - instead I'll leave it up to you to decide if the consistency going forward is worth it?

Reply:
Thanks, I wasn't aware of those other precedents. In this case, I think it's good enough reason to make it consistent, so I'll change this to "state: enabled/disabled"